packages/serverless/src/awslambda.ts

```diff
@@ -21,7 +32,8 @@ export type AsyncHandler<T extends Handler> = (
   context: Parameters<T>[1],
 ) => Promise<NonNullable<Parameters<Parameters<T>[2]>[1]>>;
 
-interface WrapperOptions {
+export interface WrapperOptions {
+  tracing: boolean;
```

Comment:
I think we can remove this option altogether 
Reason: We already have `tracingSampleRate` and `tracesSampler` that deals with enabling this.

So for example, if someone doesn't set `tracingSampleRate` to a value larger to `0` `startTransaction` does something be we never send. 
We have this function in the tracing package

```
export function hasTracingEnabled(options: Options): boolean {
  return 'tracesSampleRate' in options || 'tracesSampler' in options;
}
```

This is how we should check if tracing is enabled.